components/Blockquote/src/index.css

```diff
@@ -0,0 +1,28 @@
+.denhaag-blockquote {
+  border-inline-start: 4px solid var(--denhaag-color-green-1, #d5e7d4);
+  padding-block-start: 1rem;
+  padding-block-end: 1rem;
+  padding-inline-start: 1rem;
+  padding-inline-end: 1rem;
+}
+
+.denhaag-blockquote__content p {
+  color: var(--denhaag-color-green-4, #1d6b34);
+  font-family: var(--denhaag-typography-sans-serif-alternate-font-family, inherit);
+  font-size: var(--denhaag-typography-scale-lg, 1.25rem);
+  font-weight: 700;
+  line-height: 1.3;
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+.denhaag-blockquote__content .denhaag-blockquote__attribution {
```

Comment:
This CSS selector doesn't need to be scoped, since it's prefixed with `.denhaag-blockquote__`. Also i would suggest using the `citer` or `quoter` since it's shorter and says more about the element.